src/components/ImageView/index.native.js

```diff
@@ -55,6 +55,14 @@ class ImageView extends PureComponent {
         this.state.interactionPromise = InteractionManager.runAfterInteractions(() => this.calculateImageSize());
     }
 
+    componentDidUpdate(prevProps) {
+        // This resizes the images on cycling
```

Comment:
Let's use the same comment we did in `index.js`
```suggestion
        // Resize the image when cycled through
```